btcjson/btcdextcmds.go

```diff
@@ -89,6 +89,41 @@ func NewGetCurrentNetCmd() *GetCurrentNetCmd {
 	return &GetCurrentNetCmd{}
 }
 
+// AddMiningAddrCmd defines the setminingaddr JSON-RPC command.
+type AddMiningAddrCmd struct {
```

Comment:
Fixed. Must've been left over from the name change from `setminingaddr` to `addminingaddr`.